actionsystemtest/lib/action_system_test/test_helper.rb

```diff
@@ -0,0 +1,17 @@
+require "capybara/dsl"
+require "action_system_test/test_helpers"
+
+module ActionSystemTest
+  module TestHelper # :nodoc:
+    include TestHelpers::ActiveJobSetup
+    include TestHelpers::Assertions
+    include TestHelpers::ScreenshotHelper
+    include Capybara::DSL
+
+    Capybara.app = Rack::Builder.new do
```

Comment:
It's a bit confusing that this configuration, which is *global configuration* is set inside a module. It makes it look as though this config only applies to this module, when it in fact applies to everthing.